tests/test_utilities.py

```diff
@@ -3,13 +3,14 @@
                              create_resistive_wall_single_layer_approx_component,
                              create_resistive_wall_single_layer_approx_element,
                              create_taper_RW_approx_component,
-                             create_taper_RW_approx_element,
-                             _zlong_round_single_layer_approx,
-                             _zdip_round_single_layer_approx)
+                             create_taper_RW_approx_element)
+from xwakes.wit.utilities import (_zlong_round_single_layer_approx,
+                                  _zdip_round_single_layer_approx)
 from test_common import relative_error
 from pywit.parameters import *
-from pywit.interface import (Layer, _IW2DInputBase, FlatIW2DInput,
-                             RoundIW2DInput, Sampling, component_names)
+from pywit.interface import (FlatIW2DInput, RoundIW2DInput, Sampling,
```

Comment:
shouldn't it be `from xwakes.wit.interface import ...` (same for the line before and the one after)?

Reply:
It can also be and it will have to be the way you say when we stop supporting the `pywit` imports. On the other hand this automatically tests that the old `pywit` imports still work (i.e. that the machinery we put in place works) so maybe it is not a bad idea to keep it as it is

Reply:
I agree, why not, at this stage (we could also specifically check this behaviour in proper tests, but maybe that's being too pedantic...)